.eslintrc.json

```diff
@@ -7,18 +7,12 @@
     "browser": true,
     "es6": true
   },
-  "settings": {
-    "react": {
-      "version": "16.2.0"
-    }
-  },
   "globals": {
     "React": "readonly",
     "googlesitekit": "readonly",
     "googlesitekitAdminbar": "readonly",
     "googlesitekitDashboard": "readonly",
     "googlesitekitSettings": "readonly",
-    "lodash": "readonly",
     "googlesitekitCurrentModule": "readonly",
     "gtag": "readonly",
     "process": "readonly"
```

Comment:
Can't we remove a few of those that we no longer need?
* React (we shouldn't ever have this as a global)
* `googlesitekitDashboard` (doesn't exist?)
* `googlesitekitSettings` (doesn't exist?)
* `process` (what is this?)

Reply:
Good call. We can actually remove all of these; I've done so in a single commit (bc167dfdf47265c3bebc2ab2671235d120eadba7) in case we want to move it out of this PR, but it was easy to do and everything still works 😄